site-src/guides/use-cases.md

```diff
@@ -0,0 +1,130 @@
+      - type: Exact
+        name: X-Gateway-Model-Name
+        value: recommender
+      path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - name: deepseek-r1
+      kind: InferencePool
+      
+```
+
+
+## Example 2:   Serve LoRA adapters on a shared accelerator
+A company wants to serve LLMs for document analysis and focuses on audiences in multiple languages, such as English and Spanish. 
+They have fine-tuned models for each language, but need to efficiently use their GPU and TPU capacity. 
+You can use Gateway API Inference Extension to deploy dynamic LoRA fine-tuned adapters for each language (for example, `english-bot` and `spanish-bot`) on a common base model and accelerator. 
+This lets you reduce the number of required accelerators by densely packing multiple models on a common accelerator.
+
+The following diagram illustrates how Gateway API Inference Extension serves multiple LoRA adapters on a shared accelerator.
+![Serving LoRA adapters on a shared accelerator](../images/serve-LoRA-adapters.png)
+This example illustrates a conceptual example regarding how you can densely serve multiple LoRA adapters with distinct workload performance objectives on a common InferencePool.
+Assuming we have created container pods with our base model and model server named “gemma2b-vllm”. We instantiate an `InferencePool` resource to logically represent all the container pods that are running this base model across all the nodes in our Kubernetes cluster.
+```shell
+apiVersion: gateway.networking.x-k8s.io/v1alpha1
+kind: InferencePool
+metadata:
+  name: gemma3
+  modelServerSelector:
+  - pool: gemma3
+
+```
+Let us say we have a couple of LoRA adapters named “english-bot” and “spanish-bot” for the Gemma3 base model. 
+You can create an `InferenceModel` resource and associate these LoRA adapters to the relevant InferencePool resource.  
+In this case, we associate these LoRA adapters to the “gemma3 InferencePool resource created above.
+
+```shell
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: english-bot
+spec:
+  modelName: google/gemma-3-4b-it
+  criticality: Standard
+  poolRef:
+    name: gemma3
+        
+---
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: spanish-bot
+spec:
+  modelName: google/gemma-3-4b-it
+  criticality: Critical
+  poolRef:
+    name: gemma3
+    
+```
+Now, you can route your requests from the gateway using the `HTTPRoute` object.
+```shell
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: inference-gateway
+spec:
+  listeners:
+  - protocol: HTTP
+    port: 80
+    name: http
+
+---
+kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1
+metadata:
+  name: routes-to-llms
+spec:
+  parentRefs:
+    - name: inference-gateway
+  rules:
+  - matches:
+      path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - name: gemma3
+      kind: InferencePool
+
+```
```

Comment:
In the last, add two curl commands to demonstrate clients can now make requests to 2 different adapters, e.g.,


```
curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
"model": "english-bot",
"prompt": "What's the color of the sky",
"max_tokens": 100,
"temperature": 0
}'
```

Reply:
Do the same for example 1, e.g.:
```sh
curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -H 'X-Gateway-Model-Name: chatbot' -d '{ "model": "chatbot", "prompt": "What's the color of the sky", "max_tokens": 100, "temperature": 0 }'
```

Reply:
done